easybuild/easyconfigs/d/DL_POLY_Classic/DL_POLY_Classic-1.9-foss-2015b-PLUMED-2.1.4.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'MakeCp'
```

Comment:
the existing easyblock for DL_POLY_Classic needs an update?

seems like you're reinventing the wheel here...